admin-dev/themes/default/scss/controllers/_modules.scss

```diff
@@ -157,15 +157,15 @@ td.module_inactive {
 }
 
 .module-badge-popular {
-  background-color: $brand-addons;
+  background-color: var(--#{$cdk}blue-500);
```

Comment:
The idea behind these variables was to have a single source of truth, which comes from the UI Kit. However, as I mentioned in a previous comment, this is just the first step towards potential customization. If we want to go further, we will need to create design tokens and use them instead of generic variables. I'm available to discuss this with you.